custom-snippets/01-02-menu1/Dockerfile

```
@@ -0,0 +1,18 @@
+# Starting from a minimalist image
+FROM ruby:2.7
+# Reference for help contact me
+LABEL maintainer="chris@adadev.org"
+
+# Create a directory for the app
+RUN mkdir /app
+
+# Set the working directory for RUN, ADD and COPY
+WORKDIR /app
+
+COPY ./Gemfile .
+RUN gem install bundler
+RUN bundle install
+
+COPY . .
```

If you're going to copy everything I don't think you need to copy Gemfile individually.  (I would prefer to specify what you would like to copy individually (leaving copying Gemfile above) since that seems safer though.

```suggestion
COPY *.rb .
COPY submission.txt .
COPY test.sh .
```